examples/core/core_smooth_pixelperfect.c

```diff
@@ -69,18 +69,18 @@ int main(void)
         rotation += 60.0f*GetFrameTime();   // Rotate the rectangles, 60 degrees per second
 
         // Make the camera move to demonstrate the effect
-        cameraX = (sinf(GetTime())*50.0f) - 10.0f;
-        cameraY = cosf(GetTime())*30.0f;
+        cameraX = (sinf((float)GetTime())*50.0f) - 10.0f;
+        cameraY = cosf((float)GetTime())*30.0f;
 
         // Set the camera's target to the values computed above
         screenSpaceCamera.target = (Vector2){ cameraX, cameraY };
 
         // Round worldSpace coordinates, keep decimals into screenSpace coordinates
-        worldSpaceCamera.target.x = (int)screenSpaceCamera.target.x;
+        worldSpaceCamera.target.x = truncf(screenSpaceCamera.target.x);
```

Comment:
Is it really needed to use `truncf()`?

Reply:
yes, because the result of target uses floats. 
The input is a float and the destination is a float. the use of the int cast here is just to truncate, thus the right thing to do is to call trunc, not cast. To use the cast we'd have to do 
```worldSpaceCamera.target.x = (float)(int)(screenSpaceCamera.target.x);```
and that loos way worse.